src/Storage/Scripting/Evaluator.cs

```diff
@@ -1,83 +1,60 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Text;
-using System.Reflection;
 using System.Text.RegularExpressions;
+using SenseNet.Configuration;
 using SenseNet.Diagnostics;
 using SenseNet.Tools;
 
 namespace SenseNet.ContentRepository.Storage.Scripting
 {
-    public class Evaluator
+    public static class Evaluator
     {
-        #region Singleton
-        private static readonly object _lock = new object();
-        private static Evaluator _current;
-        private static Evaluator Current
+        static Evaluator()
         {
-            get
-            {
-                if (_current == null)
-                {
-                    lock (_lock)
-                    {
-                        if (_current == null)
-                        {
-                            var current = new Evaluator();
-                            current.Load();
-                            _current = current;
-                            SnLog.WriteInformation("Evaluator created: " + _current);
-                        }
-                    }
-                }
-                return _current;
-            }
-        }
-        private Evaluator()
-        {
-        }
-        internal static void Reset()
-        {
-            // Need to call:
-            //   Node.Delete (need to remove in-memory entries of whole deleted subtree)
-            //   Node.ForceDelete (need to remove in-memory entries of whole deleted subtree)
-            //   Modify permissions on a Node (add, remove, copy, break etc.)
-            _current = null;
+            // preload all available evaluators
+            Load();

```

Comment:
The static constructor needs to be fault tolerant because instantiating unknown type because it can cause an error and the whole system can be crashed.